esdoc.json

```diff
@@ -10,11 +10,10 @@
   "title": "RxJS",
   "styles": ["./doc/styles/main.css"],
   "scripts": [
-    "./dist/global/Rx.KitchenSink.umd.js",
+    "./dist/global/Rx.umd.js",
     "./doc/asset/devtools-welcome.js",
     "./doc/scripts/custom-manual-styles.js",
-    "./doc/decision-tree-widget/dist/decision-tree-widget.min.js",
-    "./doc/scripts/setup-rx-script.js"
+    "./doc/decision-tree-widget/dist/decision-tree-widget.min.js"
```

Comment:
No, for visibility just in case if I miss anything. esdoc script is working on my local machine with this changes.